.storybook/3.COLORS.stories.mdx

```diff
@@ -1,20 +1,12 @@
 import { Meta } from '@storybook/addon-docs';
-import ActionaleMessage from '../ui/components/ui/actionable-message';
 import designTokenDiagramImage from './images/design.token.graphic.svg';
 
-<Meta title="Design Tokens / Color" />
+<Meta title="Foundations / Color" />
 
 # Color
 
 Color is used to express style and communicate meaning.
 
-<ActionaleMessage
-  type="warning"
-  message="We are in the process of consolidating all of our colors, making them accessible and enabling theming. Many of the colors used throughout the codebase are deprecated please follow the guide below to ensure you are using the correct colors when building MetaMask UI"
-/>
-
-<br />
-
```

Comment:
We have updated all the colors(except for box shadows) during the dark mode work so this message is out of date and can be removed